### DISCUSSION TITLE
Easy way to deploy voyager instead of lemmy-ui  (ansible playbook ?) ?

### DISCUSSION THREAD

**Discussion by tifriis208** [aeharding/voyager#541] [Q&A]
Hello,

I plan to create a lemmy instance running on rpi4 powered by solar energy. I really prefer voyager instead of lemmy-ui :)
Do yo know if there is an ansible playbook or other script to install voyager instead of lemmy ui (with nginx configuration, ...).
thanks !

**Comment by aeharding** [comment]
I don't, sorry. But I recently added documentation on deploying (without docker) here, perhaps that could be helpful:

https://github.com/aeharding/voyager#traditional-deployment
